tests/performance_tests/CMakeLists.txt

```diff
@@ -1,12 +1,27 @@
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/performance_test_basic.py ${CMAKE_CURRENT_BINARY_DIR}/performance_test_basic.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/performance_test.py ${CMAKE_CURRENT_BINARY_DIR}/performance_test.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/log_reader.py ${CMAKE_CURRENT_BINARY_DIR}/log_reader.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/read_log_data.py ${CMAKE_CURRENT_BINARY_DIR}/read_log_data.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/log_reader_tests.py ${CMAKE_CURRENT_BINARY_DIR}/log_reader_tests.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/nodeos_log_2_0_14.txt.gz ${CMAKE_CURRENT_BINARY_DIR}/nodeos_log_2_0_14.txt.gz COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/nodeos_log_3_2.txt.gz ${CMAKE_CURRENT_BINARY_DIR}/nodeos_log_3_2.txt.gz COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/genesis.json ${CMAKE_CURRENT_BINARY_DIR}/genesis.json COPYONLY)
+configure_file(performance_test_basic.py performance_test_basic.py COPYONLY)
+configure_file(performance_test.py performance_test.py COPYONLY)
+configure_file(log_reader.py log_reader.py COPYONLY)
+configure_file(read_log_data.py read_log_data.py COPYONLY)
+configure_file(log_reader_tests.py log_reader_tests.py COPYONLY)
+configure_file(nodeos_log_2_0_14.txt.gz nodeos_log_2_0_14.txt.gz COPYONLY)
+configure_file(nodeos_log_3_2.txt.gz nodeos_log_3_2.txt.gz COPYONLY)
+configure_file(block_trx_data_log_2_0_14.txt.gz block_trx_data_log_2_0_14.txt.gz COPYONLY)
+configure_file(genesis.json genesis.json COPYONLY)
+configure_file(cpuTrxData.json cpuTrxData.json COPYONLY)
+configure_file(ramTrxData.json ramTrxData.json COPYONLY)
+configure_file(userTrxDataTransfer.json userTrxDataTransfer.json COPYONLY)
+configure_file(userTrxDataNewAccount.json userTrxDataNewAccount.json COPYONLY)
```

Comment:
As previously commented, it seems like converting the `<output>` field to `.`  would make this less crufty.

Reply:
Thanks! This looks better to me.